src/joint_types.jl

```diff
@@ -131,6 +131,52 @@ function velocity_to_configuration_derivative!(q̇::AbstractVector, jt::Quaterni
     nothing
 end
 
+
+"""
+$(SIGNATURES)
+
+Compute the jacobian ``Q_v`` which maps joint velocity to configuration
+derivative for the joint type ``jt``:
+
+```math
+\\dot{q} = Q_v v
+```
+
+This updates the entries of Q_v in place.
+"""
+function velocity_to_configuration_derivative_jacobian!(Q_v::AbstractMatrix, jt::QuaternionFloating, q::AbstractVector)
+    @boundscheck size(Q_v) == (7, 6)
+    quat = rotation(jt, q)
+    Q_v[1:4, 1:3] .= velocity_jacobian(quaternion_derivative, quat)
+    Q_v[1:4, 4:6] .= 0
+    Q_v[5:6, 1:3] .= 0
+    Q_v[5:7, 4:6] .= quat
+    nothing
+end
+
+"""
+$(SIGNATURES)
+
+Compute the jacobian ``V_q`` which maps joint configuration derivative to
+velocity for the joint type ``jt``:
+
+```math
+v = V_q \\dot{q}
+```
+
+This updates the entries of V_q in place.
+"""
+function configuration_derivative_to_velocity_jacobian!(V_q::AbstractMatrix, jt::QuaternionFloating, q::AbstractVector)
+    @boundscheck size(V_q) == (6, 7)
+    quat = rotation(jt, q)
+    V_q[1:3, 1:4] .= velocity_jacobian(angular_velocity_in_body, quat)
+    V_q[1:3, 5:7] .= 0
```

Comment:
I'm considering not setting these elements and just documenting that we only touch the structural nonzeros.